src/ui.rs

```diff
@@ -426,7 +448,7 @@ impl<C> Ui<C> {
     /// the "update" stage of an event loop.
     pub fn set_widgets<F>(&mut self, user_widgets_fn: F)
         where C: CharacterCache,
-              F: FnOnce(&mut Self),
+              F: FnOnce(&mut Self)
```

Comment:
I wish it would leave the trailing in all of the where clauses as it makes it much easier to shuffle around clauses when refactoring.